scripts/analytics/benchmarks.py

```diff
@@ -217,15 +209,15 @@ def main():
     projects_forked = get_projects_forked()
     projects_registered = get_projects_registered()
 
-    number_projects = len(projects)
+    number_projects = projects.count()
```

Comment:
My python internals isn't as strong as it should be, but given that len() is super-pythonic and using it is disastrous from a performance standpoint, I wonder if we shouldn't make a maintenance ticket to override `__len__` for query sets to do what count() does.